data/overlayfiles/motd-manager-server-byos/etc/motd

```diff
@@ -0,0 +1,19 @@
+SUSE Manager {SUMA_VERSION} Server
+{OS_PRETTY_NAME} {ARCH} (64-bit)
+
+    ____
+  /@    ~-.
+  \/ __ .- |
+   // // @
+
+Before setting up SUSE Manager, please make sure the system's host name
+is configured to match its DNS name.
+
+To setup SUSE Manager please execute: sudo /sbin/yast2 susemanager_setup
```

Comment:
I thought we had some extra comment to alert the user of the setup scrip and the disk separation?

Reply:
Currently there is no wording for that on the MOTD

Do we want to add it or keep the wording as is ?

Reply:
I don't think we had that in the motd. It's in the public cloud requirements section of the SUSE Manager documentation. We can add a comment with a link, of course.

Reply:
Yes, there was not previously in the motd

Yes, I'm in favor of adding a link to that section

Reply:
Before adding that link to the motd, one can reach that section with current 
`Documentation: https://www.suse.com/documentation/suse_manager/` -> `Installation Guide` on that page (HTML or PDF) ->  choose `Requirements` -> `Public Cloud Requirements` on that page

shall we be explicit and add it besides it can be reached through the current links ?

Reply:
> no value in having different PAYG and BYOS motd versions wrt documentation links

Well, they point to different URLs, we can improve that in another release/version, 
main thing is BYOS and PAYG have the right URLs and get this image out 

> Neither the old nor the new version of the PAYG overview doc link works

Yes, I did test the links when they shared them as well, when I asked they said that 
that behaviour is expected and it will work soon

Reply:
Fine with me, assuming they will work eventually. ;-)